specification/index.md

```diff
@@ -1,48 +1,42 @@
-E-ARK CSIP Table Of Contents
-============================
-
-Version: 2.0.0
-
-May 31, 2019
 
 Contents
 --------
-- [1 Introduction](#1-introduction)
-	- [1.1 The Common Specification for Information Packages and OAIS](#11-the-common-specification-for-information-packages-and-oais)
-	- [1.2 The Common Specification for Information Packages and Content Information Type Specifications](#12-the-common-specification-for-information-packages-and-content-information-type-specifications)
-	- [1.3 Common Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#13-common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
-	- [1.4. Relation to other documents](#14-relation-to-other-documents)
+- [troduction](#introduction)
+	- [he Common Specification for Information Packages and OAIS](#the-common-specification-for-information-packages-and-oais)
+	- [he Common Specification for Information Packages and Content Information Type Specifications](#the-common-specification-for-information-packages-and-content-information-type-specifications)
+	- [ommon Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
+	- [Relation to other documents](#relation-to-other-documents)
 		- [International standards and best-practices](#international-standards-and-best-practices)
 		- [Other E-ARK specifications](#other-e-ark-specifications)
-	- [1.5. Structure of the document](#15-structure-of-the-document)
+	- [Structure of the document](#structure-of-the-document)
 - [PART I: Common Specification for Information Packages](#part-i-common-specification-for-information-packages)
-	- [2.	Need for establishing common ground](#2-need-for-establishing-common-ground)
-	- [3. Requirements for CSIP Information Packages](#3-principles-for-interoperable-information-packages)
-		- [3.1. General principles](#31-general-principles)
-		- [3.2. Identification of the Information Package](#32-identification-of-the-information-package)
-		- [3.3.	Structure of the Information Package](#33-structure-of-the-information-package)
-		- [3.4.	Information Package Metadata](#34-information-package-metadata)
+	- [eed for establishing common ground](#need-for-establishing-common-ground)
+	- [equirements for CSIP Information Packages](#principles-for-interoperable-information-packages)
+		- [General principles](#general-principles)
+		- [Identification of the Information Package](#identification-of-the-information-package)
+		- [Structure of the Information Package](#structure-of-the-information-package)
+		- [Information Package Metadata](#information-package-metadata)
```

Comment:
Do we miss letters?

Reply:
This file's to go so don't worry about it.